### ISSUE TITLE
[imaging_uploader] Data framework

### ISSUE THREAD

**Issue by xlecours** [aces/Loris#8895]
Make the module use data provisioners with applicable filters

**Comment by thedamnedrhino** [comment]
@nicolasbrossard do you have this on your roadmap?

**Comment by nicolasbrossard** [comment]
Yes I do. What's the ETA for this? Also, who's behind @thedamnedrhino ?  :)

**Comment by thedamnedrhino** [comment]
hi nic :) 
sorry i missed the notification for this. 
can we try to get it by the end of the year?
you know who i am ;)

**Comment by thedamnedrhino** [comment]
gracias monsieur
